connectors/utils.py

```diff
@@ -258,6 +259,15 @@ def get_size(ob):
     return asizeof.asizeof(ob)
 
 
+def get_base64_value(content=b""):
```

Comment:
any reason to have a default `b""` value? I would assume content is always provided

Reply:
yes, We'll always get the content but we passed this as default value.

Reply:
yes, let's remove it, please, if we always pass content.